gradoop-flink/src/main/java/org/gradoop/flink/model/api/operators/LogicalGraphOperators.java

```diff
@@ -344,6 +345,102 @@ LogicalGraph reduceOnNeighbors(
    */
   DataSet<Boolean> equalsByData(LogicalGraph other);
 
+  /**
+   * Creates a graph with the same structure but a specified property of all vertex types is drilled
+   * up by the function. The original information is also kept to be able to drill down afterwards.
+   *
+   * Drill up is a convenience method for the
+   * {@link LogicalGraph#transformVertices(TransformationFunction)} operation and can be used as
+   * pre processing for the {@link LogicalGraph#groupBy(List)} operation.
+   *
+   * @param propertyKey property key
+   * @param function drill up function
+   * @return graph with drilled up properties
+   */
+  LogicalGraph drillUpVertex(String propertyKey, DrillFunction function);
+
+  /**
+   * Creates a graph with the same structure but a specified property of a specific vertex
```

Comment:
Update for each method (and adapt for edges):

"Applies a given drill-up (transformation) function on a single vertex property of the input graph. The property is identified by the specified property key. The previous version of the property value is stored at the vertex. The structure of the graph remains unchanged.

This is a convenience function for {@link LogicalGraph#transformVertices(TransformationFunction)} and can be used as a pre-processor for {@link LogicalGraph#groupBy(List)} which performs the structural aggregation."